docs/examples/couchbase.md

```diff
@@ -0,0 +1,272 @@
+<!--[metadata]>
++++
+title = "Dockerizing a Couchbase service"
+description = "Dockerizing a Couchbase service"
+keywords = ["docker, example, package installation, networking, couchbase"]
+[menu.main]
+parent = "smn_applied"
++++
+<![end-metadata]-->
+
+# Dockerizing a Couchbase service
+
+This example shows how to start a [Couchbase](http://couchbase.com) server using Docker Compose, configure it using its [REST API](http://developer.couchbase.com/documentation/server/4.0/rest-api/rest-endpoints-all.html), and query it.
+
+Couchbase is an open source, document-oriented NoSQL database for modern web, mobile, and IoT applications. It is designed for ease of development and Internet-scale performance.
+
+## Start Couchbase server using Docker Compose
+
+Couchbase Docker images are published at [Docker Hub](https://hub.docker.com/_/couchbase/).
+
+Use the following Compose file to start Couchbase server:
+
+```
+mycouchbase:
+  image: couchbase/server
```

Comment:
Maybe a dumb question : why `couchbase/server` instead of the official image `couchbase` ? (https://hub.docker.com/_/couchbase/)